cmd/chaos-builder/main.go

```diff
@@ -48,20 +48,29 @@ const codeHeader = `// Copyright 2020 Chaos Mesh Authors.
 // limitations under the License.
 
 package v1alpha1
+`
 
+func main() {
+	generatedCode := codeHeader + `
 import (
 	"reflect"
 	"time"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
-
 `
+	generatedTest := codeHeader + `
+import (
```

Comment:
Can we define these imports as a const and avoid to define these in main function? such as `const testImport = `xxxx`

Reply:
Fixed